src/resources/js/dependency.js

```diff
@@ -306,7 +305,7 @@
 	}, obj.selectors.dependency );
 
 	// Configure on Document ready for the default trigger
-	$document.ready( obj.setup );
```

Comment:
Why the modification here? It seems to be the same no?

Reply:
Because most common/existing ways of using `$.ready` are now deprecated - it's back-compatible but forward-looking.
See: https://api.jquery.com/ready/#ready-handler Starting with "jQuery offers several ways to attach a function that will run when the DOM is ready."

Reply:
Hun crazy that they deprecated the `.ready()` method.

We will need to do a major change in our codebase.

Reply:
At some point, yeah. This is a a small one (I think) for now?